File: src/ui/app/root/application/models/Setup/AssessmentType.ts
export interface IAssessmentType {

	assessmentTypeId : string;
	assessmentCategoryId : string;
	examTypeId : string;
	assessmentCategory : string;
	assessmentType : string;
	code : string;
	statusId : number;


}
export interface IAssessmentTypeAdd {
	assessmentTypeId : string;
	assessmentCategoryId : string;
	examTypeId : string;
	code : string;
	statusId : number;


}
// export interface IAssessmentTypeAddEx {
// 	assessmentTypeId : string;
// 	assessmentCategoryId : string;
// 	examTypeId : string; 
// 	statusId : number;

// }
export interface IAssessmentCategory {
	assessmentCategoryId : string;
	FullName : string;
	Code : string;
	statusId : number;


}

File: src/data/Model/Cms360/Assessment/AssessmentType.cs
/*
*   Auther: H.Muhammad Kamran
*   email: hmuhdkamran@gmail.com
*   contact: +92 (313 / 333) 9112 845
*/

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model
{
  [Table("AssessmentType", Schema = "Assessment")]
  public partial class AssessmentType
  {
    [Key]
    [Required]
    public Guid AssessmentTypeId { get; set; }

    public Guid AssessmentCategoryId { get; set; }
    public Guid ExamTypeId { get; set; }
    public int StatusId { get; set; }

  }
  [Table("VWAssessmentTypeData", Schema = "Assessment")]
  public partial class AssessmentTypeVM
  {
    [Key]
    [Required]
    public Guid AssessmentTypeId { get; set; }
    public Guid AssessmentCategoryId { get; set; }
    public Guid ExamTypeId { get; set; }

    public string AssessmentCategory { get; set; }
    public string AssessmentType { get; set; }
    public string Code { get; set; }

    public int StatusId { get; set; }

  }

  // [Table("AssessmentCategory", Schema = "Assessment")]
  // public partial class AssessmentCatorgorydata
  // {
  //   [Key]
  //   [Required]
  //   public Guid AssessmentCategoryId { get; set; }
  //   public string FullName { get; set; }
  //   public string Code { get; set; }

  //   public int StatusId { get; set; }

  // }

}